src/js/components/DataFilter/__tests__/__snapshots__/DataFilter-test.tsx.snap

```diff
@@ -583,8 +583,8 @@ exports[`DataFilter children 1`] = `
 .c7 {
   margin-left: 0px;
   margin-right: 0px;
-  margin-top: 6px;
-  margin-bottom: 6px;
+  margin-top: 12px;
```

Comment:
Changes only due to jsdom, does not appear in browser. Correct value is re-assigned below